app/views/recipe_foods/new.html.erb

```diff
@@ -0,0 +1,12 @@
+  <body>
+    <div class='container'>
+      <h2>New Ingredient:</h2>
+      <%= form_with model: @recipefoods, url: ingredient_create_path, method: :post do |f| %>
+        <%= f.label 'Ingredient' %>
+        <%= f.select :food_id, options_for_select(@foods.collect { |food| [ food.name, food.id ] }) %>
+        <%= f.label "Quantity:" %>
+        <%= f.number_field :quantity, min: 1 %>
+        <%= f.submit "Add Ingredient" %>
+      <% end %>
```

Comment:
- [ ] @asdt560 and @d4nQw3rty good work on your views however its best practice to use partial views for your form, kindly add it as you did for the foods.